netci.groovy

```diff
@@ -470,6 +414,66 @@ branchList.each { branchName ->
     }
 }
 
+// adds steps to a job to download coreclr artifacts and corefx artifacts and invoke run-test.sh.
```

Comment:
> I personally would prefer such runs in the coreclr repo itself. I don't like the cross repo dependencies being this coupled

I'm not understanding.  Either a corefx run needs to copy over coreclr artifacts (the runtime) or coreclr needs to copy over corefx artifacts (all of the libraries and all of the tests).  Why is one worse than the other?

> and I would prefer corefx never have to build coreclr during our CI.

This isn't building coreclr, it's just copying over recently built bits from Jenkins.